PetFeeding/src/main/java/com/epam/dao/PersonDao.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.dao;
+
+import com.epam.domain.Person;
+
+import java.math.BigInteger;
+
+public interface PersonDao extends Dao<Person, BigInteger> {
```

Comment:
Integer or Long is enough, you don't need BigInt

Reply:
fixed